app/services/puppet_class_importer.rb

```diff
@@ -179,7 +179,7 @@ def proxy_environments
   end
 
   def to_be_created_environments
-    proxy_environments - Environment.where(:name => proxy_environments).pluck(:name)
+    proxy_environments - Environment.unscoped.where(:name => proxy_environments).pluck(:name)
```

Comment:
Why is this necessary? With this, no error is shown to the user - just "No new environments created". Before, this would attempt to create a duplicate environment - but it would show the error "Name already exists (even if not visible to current user)"